pages/common/rails-db.md

```diff
@@ -0,0 +1,35 @@
+# rails db
+
+> Rails database actions.
+
+- Access database console:
+
+`rails db`
+
+- Create databases defined in current environment:
+
+`rails db:create`
+
+- Destroy databases defined in current environment:
+
+`rails db:drop`
+
+- Run pending migrations:
+
+`rails db:migrate`
+
+- View status of each migration file:
+
+`rails db:migrate:status`
+
+- Rollback the last migration:
+
+`rails db:rollback`
+
+- Fill database with data defined in db/seeds.rb:
```

Comment:
Perhaps this will read better?
```suggestion
- Fill the database with data defined in db/seeds.rb:
```

If _Ruby on Rails_ supports multiple databases as you've alluded to above, how does this command know which database to seed?

Reply:
Depends on the environment variable `RAILS_ENV`. By default, it's set to `development`. Perhaps I should have included this information for clarity.

Reply:
Ah, I see. Maybe saying something like this would be helpful then?

```suggestion
- Fill the current database with data defined in db/seeds.rb:
```